### ISSUE TITLE
Get invoices returns 404 error

### ISSUE THREAD

**Issue by map7** [XeroAPI/xero-ruby-oauth2-starter#8]
I'm getting an error when clicking on 'Get invoices'. I've setup my app in the developer backend and filled in the credentials in .env, I can connect and get detailed information about my tokens, but when clicking on 'Get invoices' I get the following error;

![image](https://user-images.githubusercontent.com/8230/159615555-bfcca94b-bef5-42e4-8ad7-4482f45ba8de.png)

**Comment by SerKnight** [comment]
Hey @map7 - at a glance it looks like the app assumes you can make the `/organisations` call which requires the `accounting.settings` scope to be added and active on the access_token.

Refer to https://github.com/XeroAPI/xero-ruby-oauth2-starter/blob/master/sample.env

**Comment by map7** [comment]
I do have `accounting.settings` in my scope config (.env file) and on the home page I can see that these scopes are active;

` scope: openid profile email accounting.transactions accounting.settings offline_access`

I also get an error when clicking 'Get organisation'

**Comment by map7** [comment]
I notice that the webapp sample works so I've been comparing the sinatra app to the rails app. At line 9639 of accounting_api.rb I get different results for the new_options, not sure this matters.

Good from webapp

            : {:statuses=>["AUTHORISED"], :where=>"AmountDue >0", :if_modified_since=>"2021-03-24T12:05:04+11:00", :header_params=>{"Accept"=>"application/json", :"xero-tenant-id"=>"<my tenant id>", :"If-Modified-Since"=>"2021-03-24T12:05:04+11:00"}, :query_params=>{:where=>"AmountDue >0", :Statuses=>"AUTHORISED"}, :form_params=>{}, :body=>nil, :auth_names=>["OAuth2"], :return_type=>"Invoices"}

Bad from sinatra app

            : {:header_params=>{"Accept"=>"application/json", :"xero-tenant-id"=>"<my tenant id>"}, :query_params=>{}, :form_params=>{}, :body=>nil, :auth_names=>["OAuth2"], :return_type=>"Invoices"}

**Comment by map7** [comment]
I've fixed it but don't know why this works. 

If I split the invoices line into two calls it works like so;


```
  tenant_id=xero_client.connections[0]['tenantId']
  @invoices = xero_client.accounting_api.get_invoices(tenant_id).invoices
```

But switching it back to the original doesn't work;

```
  @invoices = xero_client.accounting_api.get_invoices(xero_client.connections[0]['tenantId']).invoices
```

The same thing fixes organisations listing.

**Comment by map7** [comment]
The following pull request fixed my issue
https://github.com/XeroAPI/xero-ruby-oauth2-starter/pull/7

**Comment by RettBehrens** [comment]
Apologies for the inconvenience caused here - I've merged #7 to resolve going forward
